client/src/components/Workflow/Editor/LintSection.vue

```diff
@@ -1,22 +1,31 @@
 <template>
-    <div class="lint-section">
-        <b-button
-            :aria-expanded="expanded ? 'true' : 'false'"
-            :aria-controls="collapseId"
-            :variant="variant"
-            @click="expanded = !expanded"
-        >
-            <font-awesome-icon icon="check" v-if="okay" />
-            <font-awesome-icon icon="times" v-else />
-            {{ title }}
-            <font-awesome-icon icon="angle-double-up" v-if="expanded" />
-            <font-awesome-icon icon="angle-double-down" v-else />
-        </b-button>
-        <b-collapse :id="collapseId" v-model="expanded">
-            <div class="lint-section-body">
-                <slot> </slot>
+    <div class="mb-2">
+        <div v-if="isOkay">
+            <font-awesome-icon icon="check" class="text-success" />
+            <span>{{ successMessage }}</span>
+        </div>
+        <div v-else>
+            <font-awesome-icon icon="exclamation-triangle" class="text-warning" />
+            <span>{{ warningMessage }}</span>
+            <div v-if="hasWarningItems" class="mt-2">
+                <div
+                    v-for="(item, idx) in warningItems"
+                    :key="idx"
+                    @mouseover="onMouseOver(item.stepId)"
+                    @mouseleave="onMouseLeave(item.stepId)"
+                    class="ml-2"
+                >
+                    <a href="#" @click="onClick(item.stepId, item)" class="scrolls">
```

Comment:
```vue
                                        <font-awesome-icon
                                            v-if="input.canExtract"
                                            icon="cog"
                                            v-b-tooltip.hover
                                            title="Attempt to create a workflow input connected to this step input."
                                            @click="extractWorkflowInput(input)"
                                        />
```

The previous version of this would only show an icon if it is auto-fixable - ``v-if="input.canExtract"``. Can you modify this LintSection interface to put an auto-fixable attribute on the items?

Reply:
Showing a 'magic' icon instead of 'search' if issue is autofixable.